airflow/www/views.py

```diff
@@ -2288,6 +2292,53 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_repo = git_hash = None
+        try:
+            p = subprocess.Popen('pip freeze | grep airflow',
```

Comment:
In the git case, I load the repo based on where $AIRFLOW_HOME points. That pointer is authoritative. In terms of the PYTHONPATH and venv cases, I'd love some examples. I personally run off a git fork and use `python setup.py install` to install the code. However, many in the community don't maintain forks and instead using pip to install 

Consider that we have users and developers. Most users tend to install using standard package managers. Today, since our user documentation points to pip and also since pip plays better with automation (e.g. easy to roll forward and backward), I don't think pip will go away. 



-s